parquet/src/main/java/org/apache/iceberg/parquet/ParquetBloomRowGroupFilter.java

```diff
@@ -120,6 +124,9 @@ private boolean eval(
         return ROWS_MIGHT_MATCH;
       }
 
+      LOG.info(
+          "Bloom Filters are used. The following fields have bloom filters : {}",
+          fieldsWithBloomFilter);
       return ExpressionVisitors.visitEvaluator(expr, this);
```

Comment:
What about an empty line before the return?

Reply:
This is OK now since I moved the log inside the `if` block